src/components/swap/SwapDetailsDropdown.tsx

```diff
@@ -131,6 +159,12 @@ export default function SwapDetailsDropdown({
   const theme = useTheme()
   const { chainId } = useWeb3React()
   const [showDetails, setShowDetails] = useState(false)
+  const [isFirstPriceFetch, setIsFirstPriceFetch] = useState(true)
+  const lpFeePercent = trade ? computeRealizedLPFeePercent(trade) : undefined
```

Comment:
if this is computed from trade, can't this be computed in `formatAnalyticsEventProperties`?

Reply:
yes! sorry mistakenly thought computeRealizedLPFeePercent was a custom react hook and had to be used in the component (but it's not). moved the calc to formatAnalyticsEventProperties.